modules/460-log-shipper/hooks/internal/vector/source/kubernetes.go

```diff
@@ -67,17 +68,22 @@ type KubernetesAnnotationFields struct {
 	PodOwner       string `json:"pod_owner,omitempty"`
 }
 
+type NodeAnnotationFields struct {
+	NodeLabels string `json:"node_labels,omitempty"`
```

Comment:
New metadata field should also be added to the doc 
https://deckhouse.io/documentation/v1/modules/460-log-shipper/#kubernetes